config_defaults_inc.php

```diff
@@ -976,6 +976,18 @@
 	$g_show_avatar_threshold = DEVELOPER;
 
 	/**
+	 * To host your own avatar site change this setting.
+	 * A https url is needed if MantisBT is being accessed via https
+	 * otherwise there will be warnings of unsecure content.
+	 * @global string $g_show_avatar_url
+	 */
+	if ( http_is_protocol_https() ) {
+		$g_show_avatar_url = 'https://secure.gravatar.com';
+	} else {
+		$g_show_avatar_url = 'http://www.gravatar.com';
```

Comment:
I wouldn't rely on that, I'm planning to get rid of that variable in config_default_inc.php.